gemsfx/src/main/java/com/dlsc/gemsfx/templatepane/TemplatePane.java

```diff
@@ -32,12 +32,22 @@ public TemplatePane() {
 
 	private final ObservableMap<Position, Tile> tilesMap = FXCollections.observableHashMap();
 
+        /*
+        AR: ermöglicht das Entfernen von Nodes aus der TemplatePane.
+        */
 	public final Tile setNode(Position pos, Node node) {
 		Tile tile = tilesMap.get(pos);
-		tile.setNode(node);
-		getChildren().add(node);
-		node.setManaged(false);
-		return tile;
+                if (node!=null) {
+            		tile.setNode(node);
```

Comment:
Formatting seems invalid. Please use "spaces", not "tabs".

Reply:
OK